doc/governance.rst

```diff
@@ -48,8 +48,25 @@ Every new triager will be announced in the mailing list.
 Triagers are welcome to participate in `monthly core developer meetings
 <https://github.com/scikit-learn/administrative/tree/master/meeting_notes>`_.
 
+.. _communication_team:
+
+Communication team
+-------------------
+
+Members of the communication team help with outreach and communication
+for scikit-learn. The goal of the team is to develop public awareness of
+scikit-learn, of its features and usage, as well as branding.
+
+For this, they can operate the scikit-learn accounts on various social
+networks and produce materials.
+
+Every new communicator will be announced in the mailing list.
```

Comment:
I think we can be better at giving expectations (such as this time period) for new team members. For us, this might also means being better at mentoring them during this period -- at least I can improve.